src/file_name_plugin.js

```diff
@@ -48,8 +57,8 @@ class FileNamePlugin {
   // eslint-disable-next-line class-methods-use-this
   getUsageInfo() {
     return {
-      key: 'p',
-      prompt: 'filter by a filename regex pattern',
+      key: this._config.key || 'p',
```

Comment:
check in constructor and assign there?

Reply:
Sounds like a good idea? Do you mean assigning the `this._key` and `this._prompt` as instance variables?

Reply:
Yes 🙂